src/base/memory_monitor_common.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * memory_monitor_common.hpp - common structures for memory monitoring module
+ */
+
+#ifndef _MEMORY_MONITOR_COMMON_H_
+#define _MEMORY_MONITOR_COMMON_H_
+
+#include <string>
+#include <cstdint>
+
+typedef struct memmon_mem_stat
+{
+  uint64_t init_stat;
+  uint64_t cur_stat;
+  uint64_t peak_stat;
+  uint32_t expand_count;
+} MEMMON_MEM_STAT;
+
+typedef struct memmon_server_info
+{
+  char *name;
```

Comment:
I'm not sure how this string will be used, but typically, it is used with a char array. How about using it like 'char name[DB_MAX_IDENTIFIER_LENGTH]'?

Reply:
Changed *name -> name[DB_MAX_IDENTIFIER_LENGTH]